src/payment.js

```diff
@@ -461,13 +458,13 @@ const getUnspents = (wallet, source, notify) => {
                 // .then(over(compose(lensIndex(4), lensProp('replayable')), not))
                 // .then(over(compose(lensIndex(8), lensProp('replayable')), not))
                 // uncomment for all coins replayable
-                .then(over(compose(mapped, lensProp('replayable')), not))  // REMOVE THAT (FOR TEST)
-                .then(map(Coin.fromJS)).then(addDustIfNecessary)
+                // .then(over(compose(mapped, lensProp('replayable')), not))  // REMOVE THAT (FOR TEST)
+                .then(map(Coin.fromJS)); // .then(addDustIfNecessary)
     case is(Array, source):
       return API.getUnspent(source, -1)
                 .then(prop('unspent_outputs'))
                 .then(over(mapped, scriptToAddress))
-                .then(map(Coin.fromJS)).then(addDustIfNecessary)
+                .then(map(Coin.fromJS)); // .then(addDustIfNecessary)
```

Comment:
I think it would make more sense to have the `addDustIfNeeded` chained after this function call. Will make it easier to revert changes if we have to later.